packages/api/src/router/board.ts

```diff
@@ -1098,8 +1110,7 @@ export const boardRouter = createTRPCRouter({
                 })
                 .where(
                   and(eq(sectionLayouts.sectionId, section.id), eq(sectionLayouts.layoutId, sectionLayout.layoutId)),
-                )
-                .run();
+                );
```

Comment:
.run has to stay. it is relevant for the transaction

Reply:
I just added it back. I'm not sure how this got to be removed in the first place